crates/uv/tests/run.rs

```diff
@@ -2164,3 +2164,37 @@ fn run_script_without_build_system() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn run_script_explicit() -> Result<()> {
```

Comment:
Can you add test cases for `--script` with a file that does not exist and a directory?

Reply:
Done!